library/src/nvcc_detail/hipsparse.cpp

```diff
@@ -11466,6 +11470,124 @@ hipsparseStatus_t hipsparseSpGEMM_copy(hipsparseHandle_t      handle,
+                                               const void*            beta,
+                                               hipsparseSpMatDescr_t  matC,
+                                               hipDataType            computeType,
+                                               hipsparseSpGEMMAlg_t   alg,
+                                               hipsparseSpGEMMDescr_t spgemmDescr)
+{
+    return hipCUSPARSEStatusToHIPStatus(
+        cusparseSpGEMMreuse_compute((cusparseHandle_t)handle,
+                                    hipOperationToCudaOperation(opA),
+                                    hipOperationToCudaOperation(opB),
+                                    alpha,
+                                    (cusparseSpMatDescr_t)matA,
+                                    (cusparseSpMatDescr_t)matB,
+                                    beta,
+                                    (cusparseSpMatDescr_t)matC,
+                                    computeType,
+                                    hipSpGEMMAlgToCudaSpGEMMAlg(alg),
+                                    (cusparseSpGEMMDescr_t)spgemmDescr));
+}
+#endif
+
+#if(!defined(CUDART_VERSION) || CUDART_VERSION >= 11000)
+HIPSPARSE_EXPORT
+hipsparseStatus_t hipsparseSpGEMMreuse_copy(hipsparseHandle_t      handle,
+                                            hipsparseOperation_t   opA,
+                                            hipsparseOperation_t   opB,
+                                            hipsparseSpMatDescr_t  matA,
+                                            hipsparseSpMatDescr_t  matB,
+                                            hipsparseSpMatDescr_t  matC,
+                                            hipsparseSpGEMMAlg_t   alg,
+                                            hipsparseSpGEMMDescr_t spgemmDescr,
+                                            size_t*                bufferSize5,
+                                            void*                  externalBuffer5)
+{
+    return hipCUSPARSEStatusToHIPStatus(cusparseSpGEMMreuse_copy((cusparseHandle_t)handle,
+                                                                 hipOperationToCudaOperation(opA),
+                                                                 hipOperationToCudaOperation(opB),
+                                                                 (cusparseSpMatDescr_t)matA,
+                                                                 (cusparseSpMatDescr_t)matB,
+                                                                 (cusparseSpMatDescr_t)matC,
+                                                                 hipSpGEMMAlgToCudaSpGEMMAlg(alg),
+                                                                 (cusparseSpGEMMDescr_t)spgemmDescr,
+                                                                 bufferSize5,
+                                                                 externalBuffer5));
+}
+
+#endif
+
 #if(CUDART_VERSION >= 11020)
```

Comment:
I just noticed this as well. cusparseSDDMM was only added in cusparse 11.2.2. This passed CI as I tihnk we use cusparse 11.4 there. Do you mind changing this to 11022? If a user uses cusparse 11.2.0 or 11.2.1 I think this will cause a compilation failure

Reply:
Also I think non-reuse spgemm was added in 11.0.3 but currently it uses 11000 here I think so that should be updated. We should probably do a review of our defines at some point but it is tedious to do.

Reply:
The nice part is that in the rocsparse channel I have updated colins math libraries api compatibility spread sheet for when cusparse generic routines were added so we can reference that.

Reply:
changed to 11022